backend/app/services/quiz_service.py

```
@@ -55,7 +55,11 @@ async def get_quiz(session: AsyncSession, quiz_id: int) -> Quiz:
     Raises:
         QuizNotFoundException: If quiz id not found
     """
-    db_quiz = await session.get(Quiz, quiz_id)
+    statement = (
+        select(Quiz).where(Quiz.id == quiz_id).options(selectinload(getattr(Quiz, "questions")))
```

Ruff (0.13.1)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)